Modules/Forum/classes/class.ilForum.php

```diff
@@ -602,9 +610,22 @@ public function moveThreads($thread_ids = array(), $src_ref_id = 0, $dest_top_fr
 				$visits = 0;
 				foreach ($thread_ids as $id)
 				{
-					$objTmpThread = new ilForumTopic($id);					
+					$objTmpThread = new ilForumTopic($id);
+
+					try {
+						$numPosts = $objTmpThread->movePosts(
+							$src_top_frm_fk,
+							$oldFrmData['top_pk'],
+							$dest_top_frm_fk,
+							$newFrmData['top_pk']
+						);
+					} catch (\ilFileUtilsException $exception) {
+						$this->errorHandler->raiseError(
```

Comment:
@legionth I suggest to put this message on a stack an used `\ilUtil::sendInfo();` to display a message, because you can move multiple threads, so multiple operations can fail.